networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.adoc

```diff
@@ -0,0 +1,16 @@
+[id="configuring-hybrid-networking"]
+= Configuring hybrid networking
+include::modules/common-attributes.adoc[]
+:context: configuring-hybrid-networking
+
+toc::[]
+
+As a cluster administrator, you can configure the OVN-Kubernetes default Container Network Interface (CNI) network provider to allow Linux and Windows nodes to host Linux and Windows workloads, respectively.
```

Comment:
Kind of my fault for getting started with this, but Marc Curry had suggested cluster network provider instead, because _default_ can be confusing. It's actually not the default in OCP 4.6, but might be in the future.

Here, _default_ was meant to refer to the default network as presented by Multus, which supports _additional networks_ besides the _default_ one that handles all cluster traffic.

So really it's the _OVN-Kubernetes Container Network Interface (CNI) cluster network provider_

So, oops.